lib/interactive_viewer.ml

```diff
@@ -104,6 +105,16 @@ let change_summary z_patches : ui Lwd.t =
   W.string ~attr:Notty.A.(fg lightcyan) operation_count
 
 let view (patches : Patch.t list) =
+  let help_panel =
+    Ui.vcat
+      [
+        W.string "Help Panel:\n";
+        W.string "h:   Toggle the help panel";
```

Comment:
`h` does not actually _toggle_ the help panel. Instead, one has to use `q` to get out of the help panel.

I would prefer the "toggling" mechanism advertised than the "`q` to quit help panel" currently implemented!

Reply:
@panglesd In my last PR for the same issue, I was asked to set ```q``` for exiting the help panel by @Julow. So do you want me to set ```h``` for toggling the help panel and when the help panel is open, ```q``` to do nothing ?

Reply:
Oh, yes he did say it! Sorry for the disorder.
Let's keep what @Julow told you to do! However, let's have the description match. So `h` should be described as "opening" not "toggling" the help panel.